tests/lca/imagebasedinstall/cnf/internal/cnfconfig/config.go

```diff
@@ -0,0 +1,82 @@
+package cnfconfig
+
+import (
+	"os"
+	"path/filepath"
+	"runtime"
+
+	"github.com/golang/glog"
+	"github.com/kelseyhightower/envconfig"
+	"github.com/openshift-kni/eco-gotests/tests/lca/imagebasedinstall/cnf/internal/cnfparams"
+	"github.com/openshift-kni/eco-gotests/tests/lca/imagebasedinstall/internal/ibiconfig"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	// PathToDefaultIbuCnfParamsFile path to config file with default system tests parameters.
+	PathToDefaultIbuCnfParamsFile = "./default.yaml"
+)
+
+// CNFConfig type contains cnf configuration.
+type CNFConfig struct {
+	*ibiconfig.IBIConfig
+	TargetHubKubeConfig string `envconfig:"ECO_LCA_IBI_CNF_KUBECONFIG_TARGET_HUB"`
```

Comment:
we should use the exact same env var for kubeconfigs across FT RAN test suites in my opinion, so it is possible to run multiple test suites post deployment with the same envvars.

https://github.com/openshift-kni/eco-gotests/blob/main/tests/cnf/ran/internal/ranconfig/config.go#L52 

spoke kubeconfig should just be the default. https://github.com/openshift-kni/eco-gotests/blob/main/tests/cnf/ran/internal/ranconfig/config.go#L66

Reply:
Also not sure this should stay in lca dir, if there is nothing to share in this folder. 
It seems existing ibi/siteconfig operator test cases are in different location under cnf/ran.  I would let @klaskosk to recommend the proper directory for these tests.

Reply:
I moved the test to cnf/ran/gitopsztp as per your suggestion.